puppeteer-mocha/specs/sample.spec.js

```diff
@@ -0,0 +1,43 @@
+const puppeteer = require('puppeteer');
```

Comment:
This will serve as a standalone script to run puppeteer test.
Can you add test config in different config files?

Reply:
@ankur-lt i dont think mocha supports default load from config file similar to jest, we can move these to separate files and import and use it if thats okay.